### PR TITLE
feat: added quiz and results page

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added Quiz and results page

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
Parth everything seems fine to me but I guess you forgot the categories thing

**Comment by ParthParolekar** [comment]
> Parth everything seems fine to me but I guess you forgot the categories thing

There is an option to select a quiz (https://deploy-preview-2--pensive-allen-b82f52.netlify.app/pages/category/category.html)

File: pages/quiz/quiz.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+.options {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.modal {
+  position: sticky !important;
```
Comment (vrinda-mahajan) [inline pages/quiz/quiz.css:36]: Avoid using !important as much as possible. Everything else looks awesome.

**Review by vrinda-mahajan — APPROVED** [review]

File: pages/quiz/quiz.css (hunk 1 above)
Comment (ParthParolekar) [reply pages/quiz/quiz.css:36]: I will take care of it.

**Comment by ParthParolekar** [comment]
Thank you @Dev-AkshatGupta and @vrinda-mahajan for the reviews. Merging now

**Pull request merged**